### ISSUE TITLE
[Performance] Considerably worse performance in 1.17.1 releases?

### ISSUE THREAD

**Issue by Lythael** [sp614x/optifine#6085]
My PC is nothing too good and Optifine used to help greatly with Minecraft performance.
To the point i had 60-65 fps constantly without shaders and 10 chunk render distance , 100% entity distance. Around empty open area and my fairly busy base.

And 60 fps with BSL shaders in open areas with same settings and 40~ around busy areas.

Now i struggle to get above 50 fps in open areas , same settings , no shadders. 20-30 FPS in busy areas.
With shaders .. yea i can forget about using BSL ever again it's a slide show even in empty areas.

What happened? The performance increase is very miniscule almost non existant over vanilla Minecraft now.
While on 1.16.5 It was huuuuge performance boost over vanilla (Had 50~ fps in empty areas in 1.16 vanilla too)

OptiFine Version
Any of the 1.17.1 releases.

**Comment by Whitelisted1** [comment]
Yeah, this is a problem with the optifine pre, since others have reported it as well.. No fix yet other than turning down your quality settings

**Comment by Lythael** [comment]
> Yeah, this is a problem with the optifine pre, since others have reported it as well.. No fix yet other than turning down your quality settings

I see. There is also another issue i noticed. A lot of nearby blocks flatout don't load i have to hit "Reload Chunks" hotkey often when entering my world or traversing between dimensions.

**Comment by KaiAF** [comment]
This has been reported a lot, most likely this is caused by another issue. Until then, no idea what to tell you.

**Comment by Lythael** [comment]
> This has been reported a lot, most likely this is caused by another issue. Until then, no idea what to tell you.

I tried Sodium 1.17.1 release instead .. same thing with the performance there was 0 difference between Optifine and Sodium in 1.17.1
So Minecraft going into OpenGL 3.2 is most likely the cause.

If neither Optifine nor Sodium can negate this performance loss then this is hopeless.
